packages/validation-manager/src/ValidationManager.ts

```diff
@@ -327,4 +333,147 @@ export class ValidationManager implements IValidationManager {
   async getOperationHash (userOp: OperationBase): Promise<string> {
     return await this.entryPoint.getUserOpHash(packUserOp(userOp as UserOperation))
   }
+
+  // todo fix rest of the code to work with the new tracer result instead of adjusting it here
```

Comment:
New TODO?

Reply:
Well yes. Now, that all tests pass and the changes were already made to the tracer in geth, and to the parser here, it would be easier to add the remaining changes to the parser here instead of converting the return type. Not sure we need it right now though...